ProblemsSolution.Tests/ProblemsSolutionTest.cs

```diff
@@ -0,0 +1,33 @@
+using NUnit.Framework;
+
+namespace ProblemsSolution.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+
+        [TestCase (new int[] {1,2,3,4,5}, ExpectedResult=6)]
+        [TestCase (new int[] {12, 22, 1, 3}, ExpectedResult=34)]
+        [TestCase (new int[] {3, 7, 0, 1}, ExpectedResult=0)]
+        public int SumArrayEvenNumbersTest(int[] numbers)
+        {
+            return ProblemsSolution.ArrayProblemsSolution.SumArrayEvenNumbers(numbers);
+        }
+
+        [TestCase (new int[] {12, 22, 1, 3}, ExpectedResult=new int[] {12, 22})]
+        [TestCase (new int[] {3}, ExpectedResult=new int[] {3})]
+        [TestCase (new int[] {7, 3}, ExpectedResult=new int[] {3, 7})]
+        [TestCase (new int[] {}, ExpectedResult=new int[] {})]
+        public int[] BigestTwoNumbersArrayTest(int[] numbers)
+        {
+            return ProblemsSolution.ArrayProblemsSolution.BigestTwoNumbersArray(numbers);
+        }
+        [TestCase (new int[] {1, 2, 3, 4, 5}, ExpectedResult=false)]
+        [TestCase (new int[] {1, 2, 3, 2, 5, 6}, ExpectedResult=true)]
+        [TestCase (new int[] {2, 3, 3, 7, 8, 5, 4, 7}, ExpectedResult=true)]
```

Comment:
Add this test case and run the tests again [TestCase (new int[] {1, 1, 3, 5, 5}, ExpectedResult=true)]

Reply:
> Add this test case and run the tests again [TestCase (new int[] {1, 1, 3, 5, 5}, ExpectedResult=true)]

Yeah, my solution doesn't handle that edge case I will have to think of a better one.